plugins/scaffolder/package.json

```diff
@@ -40,24 +42,29 @@
     "@rjsf/core": "^2.4.0",
     "@rjsf/material-ui": "^2.4.0",
     "classnames": "^2.2.6",
+    "clsx": "^1.1.1",
```

Comment:
hey we already had classnames right above here ... maybe just use that one instead, it's way more popular and it has basically the exact same api

Reply:
Good point, fixed 👍